docs/rules/padding-line-between-component-options.md

```diff
@@ -103,19 +103,22 @@ This rule enforces consistent format of empty lines between Vue component option
 
 </eslint-code-block>
 
-
 ## :wrench: Options
 
 ```json
 {
-  "vue/empty-line-between-options": ["error", "always" | "never"]
+  "vue/padding-line-between-component-options": ["error", "always" | "never"]
 }
 ```
 
 - `"always"` (default) ... add an empty line between options.
 - `"never"` ... remove empty lines between options.
 
+## :couple: Related Rules
+
+- [vue/padding-line-between-blocks](./padding-line-between-blocks.md)
```

Comment:
Ah, can you link from the `vue/padding-line-between-blocks` rule to this one, too, please?

Reply:
Sure, I'll do it later tonight or tomorrow morning. Thank you for the quick responses!